src/frontends/common/include/openvino/frontend/node_context.hpp

```diff
@@ -117,8 +117,45 @@ class FRONTEND_API NodeContext {
     std::string m_op_type;
 };
 
+struct NamedOutput {
+
+    NamedOutput(const Output<Node>& _port) : port(_port) {}
+    NamedOutput(const std::string& _name, const Output<Node>& _port) : name(_name), port(_port) {}
+
+    std::string name;
```

Comment:
how does this name is different from port name?

Reply:
This name is port name. Port may have this name or not, depending on the situation.